src/pages/settings/Profile/ProfilePage.js

```diff
@@ -70,10 +71,11 @@ class ProfilePage extends Component {
         super(props);
         this.state = {
             firstName: props.myPersonalDetails.firstName,
-            firstNameError: '',
+            firstNameError: false,
             lastName: props.myPersonalDetails.lastName,
-            lastNameError: '',
+            lastNameError: false,
             pronouns: props.myPersonalDetails.pronouns,
+            pronounError: false,
```

Comment:
I think you forgot to rename these to `hasFirstNameError` & `hasLastNameError` & `hasPronounError`.

Reply:
https://github.com/Expensify/App/pull/6899/files/5d2bd6d09ca0bbb875938b8846397d9fb94aa783..a214d92294adeac52d64d9fddde0f01ff942a817